app/javascript/App/bundles/CoinShow/InfoBar.tsx

```diff
@@ -112,6 +112,7 @@ class InfoBar extends React.Component<Props, {}> {
       market_cap,
       change1h,
       change24h,
+      volume24,
```

Comment:
Hm I think we should be consistent and call this `volume24h`

Reply:
Yeah there's a consistency issue there that should be resolved.